scripts/build.sh

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/env bash
 
-./scripts/mkDocs.sh \
-  && ./scripts/mkGitNix.sh
+./scripts/mkGitNix.sh
+
+git config user.email "travis@travis.org"
+git config user.name "travis"
+git add ./git.nix
+git commit --allow-empty -m "updates git.nix"
```

Comment:
Are you sure that you want to create empty commits?  Perhaps you could only create a commit if there are a non-empty set of changes

Reply:
True. just copied that from a site. In order to get the CI working right I need a PR open to trigger builds.